browser/ui/browser_commands.cc

```diff
@@ -898,6 +898,12 @@ void ScrollTabToBottom(Browser* browser) {
   contents->ScrollToBottomOfDocument();
 }
 
+void ExportAllBookmarks(Browser* browser) {
+  auto* profile = browser->profile();
+  DCHECK(profile);
+  chrome::ExportBookmarks(profile, /*export_path=*/base::FilePath());
+}
+
```

Comment:
I'm trying to build locally to see if this approach is valid but

```
cd brave-browser/src/brave/ && npm run build -- Static && npm start Static
```

has been running for several hours and still hasn't finished.

Reply:
Unless I'm mistaken, you should see an early failure with this stating `no member named 'ExportBookmarks' in namespace 'chrome'`. Where did you see that method defined?

Reply:
Thanks! I'm currently trying to figure out how to display a pop-up modal to ask users where they want their bookmarks saved, like when we export bookmarks manually by clicking the button.

Would you happen to know how to use that existing functionality here?